CMake/FindLibUSB.cmake

```diff
@@ -24,7 +24,7 @@ if ( PKG_CONFIG_FOUND )
 endif ( PKG_CONFIG_FOUND )
 
 if ( PKGCONFIG_LIBUSB_FOUND )
-  set ( LibUSB_INCLUDE_DIRS ${PKGCONFIG_LIBUSB_INCLUDE_DIRS} )
+  set ( LibUSB_INCLUDE_DIRS ${PKGCONFIG_LIBUSB_INCLUDEDIR} )
```

Comment:
s/PKGCONFIG_LIBUSB_INCLUDEDIR/PKGCONFIG_LIBUSB_INCLUDE_DIRS/
PKGCONFIG_LIBUSB_INCLUDE_DIRS is right, which points out `/usr/include/libusb-1.0`
PKGCONFIG_LIBUSB_INCLUDEDIR points out `/usr/include`